Auth/Auth.Platform.iOS/Credentials/OIDCCredential.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using Foundation;
+using OpenId.AppAuth;
+
+namespace AeroGear.Mobile.Auth.Credentials
```

Comment:
This needs documentation

Reply:
is this a duplicated impl in #23 ?

Reply:
Yep, I'm guessing @ziccardi copied it across for simpler dev, testing etc.